scripts/aws_bastion_mfa_export.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+while getopts p: flag
+do
+    case "${flag}" in
+        p) profile=${OPTARG};;
+    esac
+done
+
```

Comment:
Making sure the profile is set here?

if [ -z ${profile} ] then
echo "profile required" >&2; exit 1
fi

Reply:
Good call @davidsandilands 👍 

```
$ bash aws_bastion_mfa_export.sh - 
profile is required
```